docs/decisions/003-ecl-compliance.md

```diff
@@ -0,0 +1,121 @@
+# Compliance checks
+
+| Status        | accepted <!--becomes accepted, rejected or superseded later--> |
+| ------------- | -------------------------------------------------------------- |
+| **Proposed**  | 2020-04-30                                                     |
+| **Accepted**  | 2020-06-19                                                     |
+| **Driver**    | @planctus                                                      |
+| **Approver**  | @kalinchernev, @papegaill                                      |
+| **Consulted** | @papegaill, @emeryro, @kalinchernev, @pgingao                  |
+| **Informed**  | consumers                                                      |
+
+## Decision
+
+In order to facilitate the implementation of the ECL components while also enforcing somehow the correct usage of those, the ecl-twig library offers an ecl-compliance tool in the form of checks that can be performed on a component to test whether it respects the editorial guidelines defined for it.
+This tool uses only logic defined in twig templates in order to make the functionality available to any consumer of the library without any additional need, in a transparent way.
+
+## Context
+
+Ecl twig is a library of twig templates implementing vanilla ECL components: https://ec.europa.eu/component-library/.
```

Comment:
`Ecl` -> ECL, or something else which can make naming of the project consistent throughout the documents within the repository.